pandas/core/arrays/timedeltas.py

```diff
@@ -445,7 +445,7 @@ def _formatter(self, boxed=False):
 
         return _get_format_timedelta64(self, box=True)
 
-    def _format_native_types(self, na_rep="NaT", date_format=None):
+    def _format_native_types(self, na_rep="NaT", date_format=None, **kwargs):
```

Comment:
do we *actually* use these kwargs (i know they are on the method you are removing)?

Reply:
yes, without this we get a test failure bc `float_format` is passed

Reply:
ahh i c, as these are fairly generic, ok then.